### ISSUE TITLE
Ability to Run a "headless" Logstash

### ISSUE THREAD

**Issue by morgango** [elastic/stack-docker#16]
I would like to be able to use stack-docker to run Logstash with [Centralized Pipeline Management](https://www.elastic.co/blog/logstash-centralized-pipeline-management).  

To do this we would need to either make the `logstash.yml` file publicly available for editing **OR** automatically configure the logstash instance to use centralized pipeline management and specify the name of the pipelines that could be run, something like "script-1", "script-2" in the `xpack.management.pipeline.id` parameter (see [configuration](https://www.elastic.co/guide/en/logstash/current/configuring-centralized-pipelines.html) for more details).

I think the best approach might be to have a separate Logstash instance is correctly configured with a username, password, and predefined script names.  However, this instance might be commented out because it might cause confusion to have two different Logstash instances running.

I would really like to have this feature for demonstrations.

**Comment by leonrinkel** [comment]
This would be really helpful. Are there any updates?

**Comment by fxdgear** [comment]
@leonrinkel no. unfortunately. My wife and I recently had our first baby and was on parental leave. Just getting back into the swing of things today. So I'm catching up on email and issues across all the repos I help maintain. 

Though I have not forgotten about this feature! :)

**Comment by leonrinkel** [comment]
Congratulations 👼!
